fpmsyncd/routesync.cpp

```diff
@@ -90,11 +90,15 @@ void RouteSync::onMsg(int nlmsg_type, struct nl_object *obj)
         struct nl_addr *addr = rtnl_route_nh_get_gateway(nexthop);
         unsigned int ifindex = rtnl_route_nh_get_ifindex(nexthop);
 
-        if (addr != NULL)
+        if (addr == NULL)
         {
-            IpAddress nh(*(uint32_t*)nl_addr_get_binary_addr(addr));
-            nexthops += nh.to_string();
+            SWSS_LOG_INFO("Got empty netxhop gateway: %d\n", i);
```

Comment:
The reason is unclear. The commit is to prevent the error further propagating, based on rules
1. there must be some nexthop gateways and interfaces
2. the gateway and interface is 1:1 mapping